internal/namespaces/vpcgw/v1/testdata/test-list-gateway-type-simple.golden

```diff
@@ -0,0 +1,21 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟩🟩🟩 STDOUT️ 🟩🟩🟩️
+Types:
```

Comment:
It might be better to return only the `Types` field, given that it is the sole field returned.